lib/capybara/node/element.rb

```diff
@@ -429,17 +430,17 @@ def drop(*args)
 
       ##
       #
-      # Scroll the page or element
+      # Scroll the page or element.
       #
-      # Scroll the page or element to its top, bottom or middle
       # @overload scroll_to(position, offset: [0,0])
+      #   Scroll the page or element to its top, bottom or middle.
       #   @param [:top, :bottom, :center, :current] position
-      #   @param :offset
+      #   @param [[Integer, Integer]] offset
       #
-      # Scroll the page or current element until the given element is aligned at the top, bottom, or center of it
       # @overload scroll_to(element, align: :top)
+      #   Scroll the page or current element until the given element is aligned at the top, bottom, or center of it.
       #   @param [Capybara::Node::Element] element   The element to be scrolled into view
-      #   @param [:top, :bottom, :center] :align Where to align the element being scrolled into view with relation to the current page/element if possible
+      #   @param [:top, :bottom, :center] align Where to align the element being scrolled into view with relation to the current page/element if possible
```

Comment:
[note] I think the positions of overload methods is incorrect…

![image](https://user-images.githubusercontent.com/473530/58297831-26635300-7e14-11e9-9061-02a08a3dbc8b.png)

https://www.rubydoc.info/gems/capybara/3.20.2/Capybara/Node/Element#scroll_to-instance_method

Reply:
LGTM